assets/javascripts/discourse/components/lazy-video.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  class="{{concat @videoAttributes.providerName '-onebox lazy-video-container'}}
```

Comment:
you have the `concat-class` helper which would make this a little bit clearer and easier to write